apps/condo/domains/meter/constants/constants.js

```diff
@@ -0,0 +1,7 @@
+const COLD_WATER_METER_RESOURCE_ID = 'e2bd70ac-0630-11ec-9a03-0242ac130003'
```

Comment:
Where these ids are created in database?

Reply:
in migration file